src/codecs/jpeg/encoder.rs

```diff
@@ -360,6 +366,14 @@ impl<W: Write> JpegEncoder<W> {
         JpegEncoder::new_with_quality(w, 75)
     }
 
+    /// Add an EXIF segment to the JPEG file
+    ///
+    /// Expects a valid TIFF-formatted EXIF block as a byte vector.
+    pub fn with_exif(mut self, exif: Vec<u8>) -> JpegEncoder<W> {
```

Comment:
```suggestion
    pub fn with_exif_metadata(mut self, exif: Vec<u8>) -> JpegEncoder<W> {
```

Nit: would align it to `ImageDecode::exif_metadata`.

Reply:
Added